elixir/phoenix_live_view_dashboard.md

```diff
@@ -0,0 +1,325 @@
+  end
+
+  defp put_time_to_conf(socket) do
+    assign(socket, time_to_conf: time_to_conf())
+  end
+
+  defp time_to_conf do
+    cond do
+      time_to_elixirconf() > Timex.now ->
+        "It's Alive!! Alive!"
+      true ->
+        Interval.new(from: Timex.now, until: time_to_elixirconf()) 
+        |> Interval.duration(:seconds) 
+        |> Duration.from_seconds 
+        |> Formatter.format(:humanized)
+    end
+  end
+
+  defp time_to_elixirconf do
+    Timex.set(Timex.now("Europe/Prague"), [month: 4, day: 8, hour: 9, minute: 0, second: 0])
+  end
+end
+```
+
+And getting that on the page is done using the `live_render/2` function just like this:
+
+```elixir
+<div class="col-md-6">
+  <div class="card">
+    <div class="card-header">
+      <h4 class="card-title">Countdown To ElixirConfEU - Prague</h4>
+    </div>
+    <div class="card-body">
+      <%= live_render(@conn, ElixirConfLiveViewWeb.CountdownToElixirconf) %>
+    </div>
+  </div>
+</div>
+```
+
+### Github
+
+Cool now we at least know how long we have to wait... :) Now let's see if we can integrate some third-party data into our dashboard. As this post is about PhoenixLiveView which is officially not released yet, wouldn't it be nice to get some stats from github to see the issue count, stars, forks and the current open PRs to see how close we are to an actual release, right? 
+Getting data from github is a breeze if we use the Tentacat library, so let's add that to the mix!
+
+```elixir
+...
+  {:tentacat, "~> 1.0"},
+...
+```
+
+Ok, let's now implement the naive(I'll explain in a bit) way of fetching data and inject that into our LiveView component. Then we'll fetch the data from the repo using the `Tentacat.Repositories.repo_get/3`, which will basically get us all kinds of data / stats about a repo given the org_name and repo_name. Here is a snippet of what we'll fetch:
+
+```elixir
+def statistics do
+  token = Application.get_env(:elixir_conf_live_view, :github_api_key)
+  client = Tentacat.Client.new(%{access_token: token})
+
+  { 200, _, response} = Tentacat.Repositories.repo_get(client, "phoenixframework", "phoenix_live_view")
+  
+  stats = response.body 
+
+  %{
+    stars: stats["stargazers_count"],
+    issues: stats["open_issues_count"],
+    forks: stats["forks_count"]
+  }
+end
+```
+
+You'll notice that I am actually authenticating to fetch the data. You can actually get this data without authenticating, but you'll hit the rate limit very quickly, so that is why I chose to authenticate here. Using Tentacat, this is very easy, so that doesn't really make things more complicated at all.
+What I also wanted to do here is to add a countdown timer to the talk by Chris @ ElixirConf as I'm hoping that he might anounce an official release there... :) 
+Below is the render function in the component with the COuntdown timer to Chris' talk added. Notice how I now need to pass in `@socket` instead of `@conn` when adding a live_render call in this .leex template.
```

Comment:
COuntdown → countdown

Reply:
Would stuff like live_render look better as code? E.g. `live_render`? (This would also apply to other occurrences of filenames, extensions, etc.)